src/tools/rbd_mirror/ImageSync.cc

```diff
@@ -233,10 +212,10 @@ void ImageSync<I>::send_copy_image() {
 
   Context *ctx = create_context_callback<
     ImageSync<I>, &ImageSync<I>::handle_copy_image>(this);
-  m_image_copy_request = ImageCopyRequest<I>::create(
-    m_local_image_ctx, m_remote_image_ctx, m_timer, m_timer_lock,
-    m_journaler, m_client_meta, &m_client_meta->sync_points.front(),
-    ctx, m_progress_ctx);
+  m_image_copy_prog_ctx = new librbd::NoOpProgressContext();
```

Comment:
You will want to use the progress callback to update the sync point periodically

Reply:
sure, it was in my TODO.